### DISCUSSION TITLE
How can I add global variables default to my own codespaces?

### DISCUSSION THREAD

**Discussion by tnir** [community/community#20946] [Codespaces]
How can I add global variables default to all my own codespaces in existing and new repositories? https://github.com/settings/codespaces/secrets/new provides me to set only in existing ones and it looks pretty hard to select all existing repositories.

**Comment by FahdSeddik** [comment]
Hey! @tnir 
I think you might find an answer [here](https://docs.github.com/en/codespaces/developing-in-codespaces/default-environment-variables-for-your-codespace). Hope this helps!

**Reply by tnir** [reply]
Thanks. This is preset default variables by GitHub Codespaces. Could you see how https://github.com/settings/codespaces/secrets/new currently works?

**Reply by FahdSeddik** [reply]
The link you provided appears to give a 404 error.

**Reply by tnir** [reply]
What edition of GitHub are you using then? Team?

**Comment by FahdSeddik** [comment]
Make sure to mark it as answer if it does help, so people would see it easier.
